spyder/plugins/variableexplorer/widgets/arrayeditor.py

```diff
@@ -850,10 +855,13 @@ def current_dim_changed(self, index):
 
     @Slot()
     def accept(self):
-        """Reimplement Qt method"""
-        for index in range(self.stack.count()):
-            self.stack.widget(index).accept_changes()
-        QDialog.accept(self)
+        """Reimplement Qt method."""
+        try:
+            for index in range(self.stack.count()):
+                self.stack.widget(index).accept_changes()
+            QDialog.accept(self)
+        except RuntimeError:

```

Comment:
@goanpeca maybe a docstring with a description of the need of this except could be added?

Reply:
Will do! thanks :-)

Reply:
Fixed